src/lib/hooks/context/ThemeProvider.tsx

```
@@ -0,0 +1,166 @@
+export function ThemeProvider({
+  children,
+  defaultExperience = 'standard',
+  defaultColorMode = 'light',
+}: {
+  children: ReactNode;
+  defaultExperience?: ExperienceType;
+  defaultColorMode?: ColorMode;
+}) {
+  return (
+    <NextThemesProvider attribute="class" enableSystem defaultTheme={defaultColorMode} themes={['light','dark']}>
+      <InnerProvider children={children} defaultExperience={defaultExperience} />
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)